ivadomed/transforms.py

```diff
@@ -63,7 +89,40 @@ def __call__(self, sample):
         return sample
 
 
-class ROICrop2D(mt_transforms.CenterCrop2D):
+class NormalizeInstance(mt_transforms.NormalizeInstance):
+    """This class extends mt_transforms.Normalize"""
+    def undo_transform(self, sample):
+        return sample
```

Comment:
Here we skip the undo for the images as we no longer care about the intensities at this stage.